utils/build-script

```diff
@@ -26,30 +31,47 @@ from build_swift.build_swift import presets
 
 import six
 
-from swift_build_support.swift_build_support import (
-    diagnostics,
-    products,
-    shell,
-    targets,
-    workspace
-)
+from swift_build_support.swift_build_support import diagnostics
+from swift_build_support.swift_build_support import products
+from swift_build_support.swift_build_support import shell
+from swift_build_support.swift_build_support import targets
+from swift_build_support.swift_build_support import workspace
 from swift_build_support.swift_build_support.SwiftBuildSupport import (
     HOME,
     SWIFT_BUILD_ROOT,
     SWIFT_REPO_NAME,
-    SWIFT_SOURCE_ROOT,
-)
+    SWIFT_SOURCE_ROOT)
 from swift_build_support.swift_build_support.cmake import CMake
 from swift_build_support.swift_build_support.host_specific_configuration \
     import HostSpecificConfiguration
-from swift_build_support.swift_build_support.targets import \
-    StdlibDeploymentTarget
+from swift_build_support.swift_build_support.targets import (
+    StdlibDeploymentTarget)
```

Comment:
Can we drop the parenthesis?

Reply:
I put the parens there to avoid the ugly `\` escape.

Reply:
Ah, I see.  What about adding a trailing comma and moving the closing parenthesis to the next line?